app/views/posts/_cards.html.erb

```diff
@@ -2,38 +2,49 @@
 
 <div class="posts">
   <% @posts.each do |post| %>
-    <%= link_to post_path(post.id) do%>
-      <div class="post">
+    <div class="post">
+      <%= link_to post_path(post.id) do%>
         <div class="post-header">
           <div class="user">
             <%= image_tag('avatar-default.webp') %>
             <div>
-              <h2><%=post.user.name%> <%=post.user.lastName%></h2>
+              <h2><%= post.user.name %> <%= post.user.lastName %></h2>
```

Comment:
```suggestion
            <h2><%= "#{current_user.name} #{current_user.lastName}" %></h2>
```

Reply:
We use post.name instead of current_user.name because we want to show the name of the owner of the post, otherwise the user's name would be shown in any post. :c

Reply:
my bad, sorry, is correct the way to set the `post.user`  only consider the suggestion of  the interpolation.